packages/bot-web-ui/src/components/help-contents/flyout-help-base.jsx

```diff
@@ -78,6 +78,13 @@ const HelpBase = ({
                         text={localize('Previous')}
                         type='button'
                         is_disabled={should_previous_disable}
+                        rendertext={text =>
```

Comment:
```suggestion
                        renderText={text =>
```

Reply:
renderprops should be lowercase, otherwise there will be an error.